tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -114,8 +111,8 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 		framework.ExpectNoError(err, "failed to create port forwarder")
 		fwCancel = cancel
 		cfg = e2econfig.TestConfig
-		OperatorFeatures := map[string]bool{"AutoScaling": true}
-		cfg.OperatorFeatures = OperatorFeatures
+		// OperatorFeatures := map[string]bool{"AutoScaling": true}
```

Comment:
How about just removing these lines?

Reply:
This will make the auto scaling case here confusing